spring-cloud-stream/src/test/java/org/springframework/cloud/stream/binder/integration/SampleStreamApp.java

```diff
@@ -37,10 +41,11 @@
  * Sample spring cloud stream application that demonstrates the usage of {@link SpringIntegrationChannelBinder}.
  *
  * @author Oleg Zhurakousky
+ * @author Gary Russell
  *
  */
 @SpringBootApplication
-@EnableBinding(Processor.class)
+@EnableBinding(SampleStreamApp.PolledConsumer.class)
 @Import(SpringIntegrationBinderConfiguration.class)
 public class SampleStreamApp {
```

Comment:
Did you intent to modify this sample app?
When running it I get 
```
Caused by: java.lang.NullPointerException: null
	at org.springframework.cloud.stream.binder.DefaultPollableMessageSource.poll(DefaultPollableMessageSource.java:220) ~[classes/:na]
	at org.springframework.cloud.stream.binder.DefaultPollableMessageSource.poll(DefaultPollableMessageSource.java:1) ~[classes/:na]
	at org.springframework.cloud.stream.binder.integration.SampleStreamApp.lambda$0(SampleStreamApp.java:65) [test-classes/:na]
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:780) ~[spring-boot-2.0.0.M7.jar:2.0.0.M7]
	... 4 common frames omitted
```

Reply:
Oops - I didn't do a final test; will take a look.

Reply:
Hmmm... runs fine for me...
```
Polled payload: polled data
Handling payload: Hello
```

Reply:
wait, I may have been making some modifications during the review (to better understand) and this could be totally on me. So disregard it for now